test_to_iris_tabledap in test_to_objects.py fails for newer versions of iris

It seems that newer versions of iris (installed using `pip3 install scitools-iris`) no longer supports the method `extract_strict()` for the `CubeList` object.

The latest documentation for iris CubeList can be found here: [https://scitools-iris.readthedocs.io/en/stable/generated/api/iris/cube.html#iris.cube.CubeList.extract](https://scitools-iris.readthedocs.io/en/stable/generated/api/iris/cube.html#iris.cube.CubeList.extract)

Documentation for an older version with `extract_strict`: [https://scitools.org.uk/iris/docs/v2.4.0/iris/iris/cube.html?highlight=cubelist#iris.cube.CubeList.extract_strict](https://scitools.org.uk/iris/docs/v2.4.0/iris/iris/cube.html?highlight=cubelist#iris.cube.CubeList.extract_strict)

This leads to the following error when using `pytest`:

```bash
―――――――――――――――――――――――――――――――――――――――――――――――― test_to_iris_tabledap ―――――――――――――――――――――――――――――――――――――――――――――――――

taodata = <erddapy.erddapy.ERDDAP object at 0x7fc80e26f400>

    @pytest.mark.web
    @pytest.mark.vcr()
    def test_to_iris_tabledap(taodata):
        cubes = taodata.to_iris()

        assert isinstance(cubes, iris.cube.CubeList)
>       assert isinstance(cubes.extract_strict("depth"), iris.cube.Cube)
E       AttributeError: 'CubeList' object has no attribute 'extract_strict'

tests/test_to_objects.py:80: AttributeError
```

Will there be any issue with just using `extract()` method?

--- a/requirements-dev.txt
+++ b/requirements-dev.txt
@@ -9,7 +9,7 @@ flake8-comprehensions
 flake8-mutable
 flake8-print
 interrogate
-iris
+iris >=3
 isort
 joblib
 jupyter

--- a/tests/test_to_objects.py
+++ b/tests/test_to_objects.py
@@ -77,8 +77,8 @@ def test_to_iris_tabledap(taodata):
     cubes = taodata.to_iris()
 
     assert isinstance(cubes, iris.cube.CubeList)
-    assert isinstance(cubes.extract_strict("depth"), iris.cube.Cube)
-    assert isinstance(cubes.extract_strict("20C Isotherm Depth"), iris.cube.Cube)
+    assert isinstance(cubes.extract_cube("depth"), iris.cube.Cube)
+    assert isinstance(cubes.extract_cube("20C Isotherm Depth"), iris.cube.Cube)
 
 
 @pytest.mark.web

